user-guide/Advanced_Functionality/Databases/Configuring_dedicated_clustered_storage/Indexing_database/OpenSearch_Database/Configuring_OpenSearch_Backups.md

```diff
@@ -22,18 +22,26 @@ Examples:
 
 - ...
 
-## Preparing and configuring the repositories
+## Setting up a shared folder
 
-> [!IMPORTANT]
-> To be able to restore a snapshot later, you must go through these steps first. Without a functional repository, you will run into issues that will delay the snapshot restore.
+In this procedure, we will be using an [NFS server](#nfs-server) and [NFS client](#nfs-client) to establish a shared folder accessible by all OpenSearch nodes. This shared folder allows all nodes to store snapshot data. Use an NFS server and client as explained in [How to Set Up NFS Server and Client on CentOS 8](https://www.tecmint.com/install-nfs-server-on-centos-8/).
+
+> [!TIP]
+>
+> - If required, contact your IT department to follow the NFS server setup procedure, as this step may otherwise be challenging. Further input on networking infrastructure may also be required.
+> - It is also possible to set up an NFS share on Windows and share it with the destination Linux system. See [Windows Server 2016 as an NFS server for Linux clients](https://blog.bobbyallen.me/2018/01/18/windows-server-2016-as-an-nfs-server-for-linux-clients/).
 
-Below we are talking about NFS server and NFS clients. The NFS server is responsible of providing a shared folder accessible from all OpenSearch nodes. Why? Because that’s the way Elasticsearch snapshots work. All nodes must have access to a shared storage to be able to store the snapshot data.
+> [!NOTE]
+>
+> - As this setup might be more challenging, we recommend that you set it up in advance.
+> - Be wary of read, write, and execute rights, firewall configurations, and the state of the NFS server service during the setup. For the machines that will act as NFS servers, make sure the firewall allows NFS traffic. You can use the following command for this: *sudo ufw allow nfs*.
+> - Make sure the OpenSearch user has enough rights to the folder to read, write, and execute its contents.
```

Comment:
I have moved these alerts up as the NFS server and NFS client are mentioned here already.